src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String astring;
+    private int variable;
```

Comment:
What does `variable` mean?

Reply:
Renamed "variable" and "astring" names to "year" and "helloString" to make them clearer for understanding.